3rdparty/libgcov/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+check_submodule_not_empty(compiler-rt)
+
+apply_patches(patches compiler-rt)
```

Comment:
Why do you need to do this at all?  For other submodules we depend on, we depend on a fork that has the patches already applied.   This PR seems to take a different approach that has to re-apply the patches every time.